content/PreparedStatement/PreparedStatement.facts.md

```diff
@@ -0,0 +1,62 @@
+### Facts for Prepared Statement lesson:
+
+A `prepared statement` is used to execute an SQL statement multiple times with the same or different parameter values. The execution of a prepared statement consists of using `prepare()` and `execute()` methods. 
+
+- The `PDO::prepare()` method prepares an SQL statement for execution using the `PDOStatement::execute()` method and returns a `PDOStatement` object.
+
+- The `PDOStatement::execute()` method executes a prepared statement and returns `true` on success or `false` on failure.
+
+- The `PDOStatement::fetchAll()` method fetches a result set and returns an array of the remaining rows in a result set.
+
+Using `placeholders` instead of actual parameter values in a prepared statement is useful against SQL injection attacks because `placeholders` substitute the actual input values in an SQL statement. A `placeholder` is often indicated by a question mark `?` called `positional placeholder` or a colon followed by a variable name `:variable_name` called `named placeholder`. 
```

Comment:
The SQL Injection is just a side effect of using `placeholders`. I can't explain it clearly in here, but I'll talk to you for an analogy.